publish/npm/bin/pmg.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+const { spawn } = require("child_process");
+
+const BINARY_NAME = process.platform === "win32" ? "pmg.exe" : "pmg";
+const BINARY_PATH = path.join(__dirname, BINARY_NAME);
+
+function main() {
+  // Check if binary exists
+  if (!fs.existsSync(BINARY_PATH)) {
+    console.error(
+      `❌ Error: ${BINARY_NAME} binary not found at ${BINARY_PATH}`,
+    );
+    console.error("");
+    console.error("This usually means:");
+    console.error("  1. The installation failed or was incomplete");
+    console.error("  2. Your platform is not supported");
+    console.error(
+      `  3. Expected binary for: ${process.platform}-${process.arch}`,
+    );
+    console.error("");
+    console.error("Try reinstalling the package:");
+    console.error("  npm uninstall -g @safedep/pmg");
```

Comment:
Can we move these project specific names to a common `config.js` or a constant? 

That way, we can reuse all these scripts easily across all our tools. Think reusability for this `npm` wrapper across projects.